ci-operator/step-registry/telco5g/gather-pao/telco5g-gather-pao-commands.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ telco5g gather-pao commands ************"
+
+pao_mg_tag="4.10" # pao must-gather does not have 'latest' tag - setting 4.10 as a workaround for now.
+if [ ${PULL_BASE_REF} != "master" ]
+then
+        pao_mg_tag=${PULL_BASE_REF##release-}
+fi
+
+export KUBECONFIG=$SHARED_DIR/kubeconfig
+
+oc adm must-gather --image=quay.io/openshift-kni/performance-addon-operator-must-gather:${pao_mg_tag}-snapshot --dest-dir=${ARTIFACT_DIR}
```

Comment:
Ugh, I really wish we had a structured way to ship these must-gather images alongside their operators...